### DISCUSSION TITLE
NocoDB API performance better than Airtable and other competition [May 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2033] [Discord Threads]
> 🙋‍♂️ **Elian**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Awesome  
#  
> 🙋‍♂️ **BoMBx**  
#### Congrat! <:ScreenShot20200321at10:825353260554387557>  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968852671421349898)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
